src/FullBuild/Program.fs

```diff
@@ -49,12 +49,12 @@ let tryMain argv =
     // view
     | Command.AddView viewInfo -> Commands.View.Add viewInfo
     | Command.DropView viewInfo -> Commands.View.Drop viewInfo.Name
-    | Command.ListViews -> Commands.View.List ()

```

Comment:
static view can be expressed in term of dynamic view - which it is in this PR and this is fine.
My only concern is discarding dynamic view - and there is no point as of now removing support for this. Both can live together - it's just a matter defining what is a static view and what is a dynamic view (ie: where view definition lives). 

Otherwise, I'm convinced it's a great complementary addition to views - because people are bred with double-clicking on sln file - work should go on.